include/bbp/sonata/population.h

```diff
@@ -48,7 +48,7 @@ class SONATA_API Selection
     bool empty() const;
 
   private:
-    Ranges ranges_;
+    const Ranges ranges_;
```

Comment:
@jorblancoa: keeping this constant means we don't have to worry about `operator=` for now; in fact it pointed out that `nodes_ids_` in `ReportReader<T>::Population::get` was building a `Selection` only to `flatten` it after